test/generator/CMakeLists.txt

```diff
@@ -1,93 +1,191 @@
 ##
-# Convenience method for defining test cases in this directory.
+# Convenience methods for defining test cases in this directory.
 ##
 
-function(halide_define_aot_test NAME)
-    set(options OMIT_DEFAULT_GENERATOR)
-    set(oneValueArgs FUNCTION_NAME)
-    set(multiValueArgs GEN_DEPS EXTRA_LIBS ENABLE_IF FEATURES PARAMS GEN_TARGET GROUPS EXTRA_OUTPUTS)
+##
+# Some tests are not available when compiling for WASM.
+##
+
+set(_USING_WASM (TARGET_WEBASSEMBLY AND Halide_TARGET MATCHES "wasm"))
```

Comment:
Given that these values don't change throughout the file, it might be simpler to just write an if statement and set this value to a constant `1` or `0`

Reply:
Done